pynicotine/gtkgui/userbrowse.py

```diff
@@ -688,6 +695,13 @@ def on_select_dir(self, selection):
 
         self.set_directory(iterator.user_data)
 
+    def on_select_file(self, selection):
+
+        if selection is None:
+            return
+
+        self.num_selected_files = selection.count_selected_rows()
```

Comment:
Add `self.num_selected_files = 0` in __init__() too, below `self.selected_files = {}`